src/es/cms/utils.ts

```diff
@@ -178,4 +178,27 @@ function getDateFilter(types: Array<string> | null, filters: Array<FilterInput>)
   return []
 }
 
-export { getSearchQuery, getThemeFilter, getDateFilter }
+// Constructs the ES query to filter on subtype
+function getSubTypeFilter(filters: Array<FilterInput>) {
+  const subTypeFilter = filters.find((filter) => filter.type === FILTERS['SUBTYPE'].type)
+
+  if (subTypeFilter && subTypeFilter.values) {
+    return subTypeFilter.values
+      .map((value: any) => {
```

Comment:
Looking at the implementation `value` is expected to be of type `string`.

Reply:
This typing was not even needed, as it could be inherit from above